cmd/prometheus/main.go

```diff
@@ -282,6 +282,9 @@ func (c *flagConfig) setFeatureListOptions(logger *slog.Logger) error {
 			case "otlp-deltatocumulative":
 				c.web.ConvertOTLPDelta = true
 				logger.Info("Converting delta OTLP metrics to cumulative")
+			case "use-uncached-io":
```

Comment:
Don't love this name.

Reply:
wanted to avoid `use-direct-io` in case we decide to use sth else than Direct I/O for this.
I find `uncached` better than `unbuffered`. Any propositions?